src/NBitcoin/ChainedBlock.cs

```diff
@@ -111,19 +126,22 @@ public BlockLocator GetLocator()
                     nStep *= 2;
             }
 
-            var locators = new BlockLocator();
+            BlockLocator locators = new BlockLocator();
```

Comment:
we actualy DO use var in this specific case

`X x = new X();` -> we use `var x = new X();`

Reply:
WHAT? I don't believe my eyes...

Reply:
dude, all i'm doing is just telling what is in the policy, nothing else :D